src/vision/customVisionPrediction.js

```diff
@@ -0,0 +1,254 @@
+                "typeName" : "string",
+            }]
+        
+        };
+        return this.makeRequest({
+            operation: operation,
+            parameters: parameters,
+            headers: headers,
+            body: body
+        })
+    };
+
+
+    PredictImageAndSaveResults({parameters, headers, body}){
+        const operations= {
+            "path": "customvision/v1.0/Prediction/{projectId}/inline/image",
+            "method": "POST",
+            "headers": [{
+                "name": "Content-Type",
+                "description": "Media type of the body sent to the API.",
+                "options": [
+                    "multipart/form-data",
+                ],
+                "required": false,
+                "typeName": "string"
+            },{
+                "name": "PredictionKey",
+                "description": "Subscription key which provides access to this API. Found in your Profile.",
+                "required": true,
+                "typeName": "string"
+            }],
+            "parameters": [{
+                "name": "projectId",
+                "description": "Format - uuid. The project to evaluate against",
+                "value": null,
+                "required": true,
+                "type": "queryStringParam",
+                "typeName" : "string",
+
+            },{
+                "name": "iterationId",
+                "description": "Format - uuid. Optional. Specifies the id of a particular iteration to evaluate against. The default iteration for the project will be used when not specified.",
+                "value": null,
+                "required": false,
+                "type": "queryStringParam",
+                "typeName" : "string",
+            },{
+                "name": "application",
+                "description": "Optional. Specifies the name of application using the endpoint.",
+                "value": null,
+                "required": false,
+                "type": "queryStringParam",
+                "typeName" : "string",
+            }]
+        }
+
+        return this.makeRequest({
+            operation: operation,
+            parameters: parameters,
+            headers: headers,
+            body: body
+        })
+    };
+
+
+    PredictImageUrl({parameters, headers, body}){
+        const operations= {
+            "path": "customvision/v1.0/Prediction/{projectId}/url",
+            "method": "POST",
+            "headers": [{
+                "name": "Content-Type",
+                "description": "Media type of the body sent to the API.",
+                "options": [
+                    "application/json",
+                    "text/json",
+                    "application/xml",
+                    "text/xml",
+                    "application/x-www-form-urlencoded",
+                ],
+                "required": false,
+                "typeName": "string"
+            },{
+                "name": "PredictionKey",
```

Comment:
You can get rid of this parameter. Just hard-code it in line 190 with value `this.apiKey`